src/qt/coincontroldialog.cpp

```diff
@@ -448,12 +447,9 @@ void CoinControlDialog::updateLabels(WalletModel *model, QDialog* dialog)
 
         if (amount > 0)
         {
-            if (amount < COIN)
-                fLowOutput = true;
-
             CTxOut txout(amount, (CScript)vector<unsigned char>(24, 0));
             txDummy.vout.push_back(txout);
-            if (txout.IsDust(CTransaction::minRelayTxFee))
```

Comment:
This is just a display thing here. Our IsDust() is hardcoded to false, so the label there would never turn red.  
Our anti dust measure is the 1 doge extra feee. But that is not determined in IsDust().